.github/workflows/dart.yml

```diff
@@ -41,18 +41,24 @@ jobs:
       - name: Lint code
         run: melos run analyze
       - name: Run tests
+        run: melos test
+
+      - name: Setup Codecov
         run: |
-          mapfile -t packages < <(melos list --parsable --diff=${{ github.event.pull_request.head.sha }}~${{ github.event.pull_request.commits }}..${{ github.event.pull_request.head.sha }} --include-dependents | xargs -r -n1 basename)
-          if [ -z "$packages" ]; then exit 0; fi
-          packages_glob="$(printf ",%s" "${packages[@]}")"
-          packages_glob="${packages_glob:1}"
-          MELOS_PACKAGES="$packages_glob" melos test
+          cd /tmp
```

Comment:
You could just create the bin dir directly and cd into it instead of moving the binary afterwards.

Reply:
But then the key and checksum would also end up in our PATH which I want to avoid

Reply:
Ah right, that makes sense :+1: